crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -345,6 +345,22 @@ impl AuthorityPerpetualTables {
         Ok(self.pruned_checkpoint.get(&())?.unwrap_or_default())
     }
 
+    pub fn get_current_epoch_stable_sequence_number(
+        &self,
+        object_id: &ObjectID,
+        epoch_id: EpochId,
+    ) -> SuiResult<Option<VersionNumber>> {
+        let object_key = (epoch_id, ObjectKey::min_for_id(object_id));
+        let object = self.get_object(object_id);
+        Ok(match self.object_per_epoch_marker_table.get(&object_key)? {
```

Comment:
Can you add a comment why the two DB reads (get_object and get marker) are data-race safe?